openshift/packit-service-beat.yml.j2

```
@@ -0,0 +1,125 @@
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+---
+kind: DeploymentConfig
+apiVersion: v1
+metadata:
+  labels:
+    app: packit
+    name: packit-service-beat
+  name: packit-service-beat
+spec:
+  template:
+    metadata:
+      labels:
+        name: packit-service-beat
+        app: packit
+    spec:
+      volumes:
+        - name: packit-ssh
+          secret:
+            secretName: packit-ssh
+        - name: packit-secrets
+          secret:
+            secretName: packit-secrets
+        - name: packit-config
+          secret:
+            secretName: packit-config
+      containers:
+        - name: packit-service-beat
+          image: packit-worker:{{ deployment }}
+          env:
+            - name: DEPLOYMENT
+              value: {{ deployment }}
+            - name: APP
+              value: packit_service.worker.tasks
+            - name: REDIS_SERVICE_HOST
+              value: redis
+            - name: SENTRY_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: packit-sentry
+                  key: sentry
+            - name: POSTGRESQL_USER
+              valueFrom:
+                secretKeyRef:
+                  key: database-user
+                  name: postgres-secret
+            - name: POSTGRESQL_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: database-password
+                  name: postgres-secret
+            - name: POSTGRESQL_DATABASE
+              valueFrom:
+                secretKeyRef:
+                  key: database-name
+                  name: postgres-secret
+            - name: RUN_SERVICE
+              value: beat
+          volumeMounts:
+            - name: packit-ssh
+              mountPath: /packit-ssh
+            - name: packit-secrets
+              mountPath: /secrets
+            - name: packit-config
+              mountPath: /home/packit/.config
+          command:
+            - "/usr/bin/run_worker.sh"
+          resources:
+            requests:
+              memory: "100Mi"
+              cpu: "50m"
+            limits:
+              memory: "200Mi"
+              cpu: "100m"
```

The beat doesn't need so much memory. If I understand it correctly it'll do nothing most of the time.
```suggestion
            limits:
              memory: "80Mi"
              cpu: "50m"
```

---

When I was trying to run it with this configuration, beat had problems to start so I made the limits higher, is it okay?

---

ok